client/signup/steps/plans/index.jsx

```diff
@@ -163,31 +147,47 @@ export class PlansStep extends Component {
 		}
 
 		return (
-			<div>
-				{ errorDisplay }
-				<QueryPlans />
-				<PlansFeaturesMain
-					site={ selectedSite || {} } // `PlanFeaturesMain` expects a default prop of `{}` if no site is provided
-					hideFreePlan={ hideFreePlan }
-					isInSignup={ true }
-					isLaunchPage={ isLaunchPage }
-					intervalType={ this.getIntervalType() }
-					onUpgradeClick={ this.onSelectPlan }
-					showFAQ={ false }
-					domainName={ this.getDomainName() }
-					customerType={ this.getCustomerType() }
-					disableBloggerPlanWithNonBlogDomain={ disableBloggerPlanWithNonBlogDomain }
-					plansWithScroll={ this.state.isDesktop }
-					planTypes={ planTypes }
-					flowName={ flowName }
-					showTreatmentPlansReorderTest={ showTreatmentPlansReorderTest }
-					isAllPaidPlansShown={ true }
-					isInVerticalScrollingPlansExperiment={ isInVerticalScrollingPlansExperiment }
-					shouldShowPlansFeatureComparison={ this.state.isDesktop } // Show feature comparison layout in signup flow and desktop resolutions
-					isReskinned={ isReskinned }
-					disableMonthlyExperiment={ false }
-				/>
-			</div>
+			<ProvideExperimentData
+				name="calypso_signup_monthly_plans_default_202201_v2"
+				options={ {
+					isEligible: 'onboarding' === this.props.flowName || 'launch-site' === this.props.flowName,
+				} }
+			>
+				{ ( isLoading, experimentAssignment ) => {
+					if ( isLoading ) {
+						return <PulsingDot active={ true } />;
```

Comment:
Nit: we don't really need the `={ true }`
```suggestion
						return <PulsingDot active />;
```